utils.c

```diff
@@ -0,0 +1,31 @@
+#include "mkmimo.h"
+
```

Comment:
We should put `#include`s for `malloc`, `perror`, `abort`, etc. here, possibly moving from mkmimo_nonblocking.c.